src/Compilers/CSharp/Portable/Symbols/PropertySymbol.cs

```diff
@@ -332,6 +332,21 @@ public sealed override SymbolKind Kind
             }
         }
 
+        internal int OverloadResolutionPriority
+        {
+            get
+            {
+                if (!this.IsIndexer)
+                {
+                    return 0;
+                }
```

Comment:
Is this check necessary?

Reply:
I intentionally assert that no non-indexers call `TryGetOverloadResolutionPriority`, to try and avoid anyone calling that in a scenario that shouldn't be looking at priority. I suppose I could not do that, but I would prefer to do this, and not even try to crack or decode attributes in scenarios that shouldn't.